contracts/contracts/token/FakeNetworkToken.sol

```diff
@@ -0,0 +1,29 @@
+/*
```

Comment:
Is this contract used anywhere? I don't see anything

Reply:
Not yet, I've used it for testing.